jsonenums.go

```diff
@@ -75,6 +75,8 @@ import (
 	"path/filepath"
 	"strings"
 
+	"./utils"
```

Comment:
You need to modify your GOPATH so that it points to wherever you've cloned this repo instead of gillnet/go.